src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/DelegationsController.cs

```diff
@@ -56,30 +56,22 @@ public async Task<ActionResult> Post([FromBody] List<Rule> rules)
             {
                 return BadRequest("Invalid model");
             }
+            
+            List<Rule> delegationResults = await _pap.TryWriteDelegationPolicyRules(rules);
 
-            try
+            if (delegationResults.All(r => r.CreatedSuccessfully))
             {
-                List<Rule> delegationResults = await _pap.TryWriteDelegationPolicyRules(rules);
-
-                if (delegationResults.All(r => r.CreatedSuccessfully))
-                {
-                    return Created("Created", delegationResults);
-                }
-
-                if (delegationResults.Any(r => r.CreatedSuccessfully))
-                {
-                    return StatusCode(206, delegationResults);
-                }
-
-                string rulesJson = JsonSerializer.Serialize(rules);
-                _logger.LogError($"Delegation could not be completed. None of the rules could be processed, indicating invalid or incomplete input:\n{rulesJson}", rulesJson);
-                return StatusCode(400, $"Delegation could not be completed");
+                return Created("Created", delegationResults);
             }
-            catch (Exception e)
+
+            if (delegationResults.Any(r => r.CreatedSuccessfully))
             {
-                _logger.LogError(e, "Delegation could not be completed. Unexpected exception.");
-                return StatusCode(500, $"Delegation could not be completed due to an unexpected exception.");
+                return StatusCode(206, delegationResults);
             }
+
+            string rulesJson = JsonSerializer.Serialize(rules);
+            _logger.LogInformation("Delegation could not be completed. None of the rules could be processed, indicating invalid or incomplete input:\n{rulesJson}", rulesJson);
+            return StatusCode(400, $"Delegation could not be completed");
```

Comment:
There is a separate method for BadRequest that it is possible to use.
Also take note that the method currently in use is:
`ObjectResult StatusCode(int statusCode, object value)`
This means the string is treated as an object. There is no overload for any of these methods that takes a string. The result is that the body of the response is a simple string. Which is probably fine. Just making a note of it.

Reply:
Changed to BadRequest still uses string as when it fails here it has no meningfull reply just information on what is wrong.